tools/start_server.py

```diff
@@ -215,13 +221,34 @@ def handle_get_app_manifest_message(params):
 
 	return json.dumps(response_msg)
 
+
+def handle_start_streaming_adapter(params):
+	print("-->Handle start gstreamer adapter\r")
+	stream_endpoint = "http://localhost:8085"
+	ffmpeg_process = ffmpeg.input(params['url']).output(stream_endpoint, vcodec="vp8", format="webm", listen=1, multiple_requests=1).run_async(pipe_stderr=True) 
```

Comment:
@LuxoftAKutsan suggest to add check that `params` actually contains `url` for performing some basic validation

Reply:
bba3698